reactjs/components/atoms/Paragraph/List/List.js

```diff
@@ -37,8 +37,8 @@ class List extends React.Component {
             <Wrapper>
               {list.map((item, index) => (
                 type === 'list_checkbox' ?
-                  <Checkbox key={index} label={item}/> :
-                  <li key={index}><span>{item}</span></li>
+                  <Checkbox key={id + '_' + index} label={item} id={id + '_' + index}/> :
+                  <li key={id + '_' + index}><span>{item}</span></li>
```

Comment:
@andreityuhai great stuff! 
Finally let's extract that expression into a variable to make it clean and awesome 👍